### PR TITLE
Add documentation for Operational Monitoring.

### PR DESCRIPTION
**Pull request by emtwo**
Note that the reference [here](https://github.com/mozilla/bigquery-etl/blob/main/bigquery_etl/operational_monitoring/README.md) is still being worked on and will become less similar to this page and more of an implementation details page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ the `.spelling` file in the code repository.
 For things like dataset names or field names, `code blocks` should be preferred. Things like
 project names or common technical terms should be added to the `.spelling` file.
 
-The [markdown-spell-check](https://www.npmjs.com/package/markdown-spellcheck) package checks spelling as part of the build process. To run it locally, install [node.js](https://nodejs.org/en/) (if not already installed) and run `npm install` at the root of the repository. Then run the `scripts/link_check.sh` script.
+The [markdown-spell-check](https://www.npmjs.com/package/markdown-spellcheck) package checks spelling as part of the build process. To run it locally, install [node.js](https://nodejs.org/en/) (if not already installed) and run `npm install` at the root of the repository. Then run the `scripts/spell_check.sh` script.
```
Comment (badboy) [inline README.md:67]: 🙈

**Comment by emtwo** [comment]
@badboy Would you like to review this PR since you seem to have looked at it already? 😁🙃

File: src/cookbooks/operational_monitoring.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
+# Introduction to Operational Monitoring
+
+Operational monitoring is a general term that refers to monitoring the health of software.
+
+However, Mozilla's internal Operational Monitoring tooling supports a couple of specific use cases:
+
+1. Monitoring build over build. This is typically for Nightly where one build may contain changes that a previous build doesn't and we want to see if those changes affected certain metrics.
+2. Monitoring by submission date over time. This is helpful for a rollout in Release for example, where we want to make sure there are no performance or stability regressions over time as a new build rolls out.
+
+The monitoring dashboards produced for these use cases are available in [Looker](https://looker.com/).
```
Comment (scholtzan) [inline src/cookbooks/operational_monitoring.md:10]: We can link directly to the operational monitoring folder in Looker:
```suggestion
The monitoring dashboards produced for these use cases are available in [Looker](https://mozilla.cloud.looker.com/folders/494).
```

File: src/cookbooks/operational_monitoring.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+Access to the Looker Operational Monitoring dashboards is currently limited to Mozilla employees and designated contributors. For more information, see [gaining access](../concepts/gaining_access.md).
+
+## How to use Operational Monitoring
+
+An operational monitoring project is defined using a project definition JSON file in combination with 2 other JSON files.
+
+In order to create a new project to be monitored:
+
+1. A new `<project_name>.json` file must be added to the `operational_monitoring` GCS bucket at `gs://operational_monitoring/projects`.
```
Comment (scholtzan) [inline src/cookbooks/operational_monitoring.md:20]: I remember having trouble finding the right project that has the bucket at some point. Might be worth adding what GCP project the bucket is in

**Comment by badboy** [comment]
> @badboy Would you like to review this PR since you seem to have looked at it already? 😁🙃

I can give it a proper look, yes.

File: src/cookbooks/operational_monitoring.md (hunk 2 above)
Comment (badboy) [reply src/cookbooks/operational_monitoring.md:10]: Sidenote: I seem to not have access to data in the Fission dashboard. Are those intentionally locked down?

File: src/cookbooks/operational_monitoring.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+  },
+  "os": {
+    "source": ["mozdata.telemetry.main_nightly", "mozdata.telemetry.crash"],
+    "sql": "normalized_os"
+  }
+  ...
+}
+```
+
+Note that each key in `dimensions.json` is the name of a dimension that will appear as a dropdown in the generated dashboards and can be referenced in the `<project_name>.json` file below. For each dimension, the following fields exist:
```
Comment (badboy) [inline src/cookbooks/operational_monitoring.md:43]: I think I would start the section with an explanation of what this is first.
So moving this sentence above the JSON example (and dropping the "Note that")

File: src/cookbooks/operational_monitoring.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+    "source": "mozdata.telemetry.main_nightly",
+    "category": "performance",
+    "type": "histogram",
+    "sql": "payload.histograms.content_process_count"
+  },
+ ...
+}
+```
+
+Note that each key in `probes.json` is the name of a probe that will be used in the generated dashboards and can be referenced in the `<project_name>.json` file below. For each probe, the following fields exist:
```
Comment (badboy) [inline src/cookbooks/operational_monitoring.md:68]: Same as above: move the explanation above the example.

Comment (badboy) [reply src/cookbooks/operational_monitoring.md:68]: (strong opinion, loosely held. If you feel this way around is better I'm ok with that)

File: src/cookbooks/operational_monitoring.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+
+- `source`: The BigQuery table name to be queried.
+- `category`: This can be any string value. It's currently not being used but in the future, this could be used to visually group different probes by category.
+- `type`: This is used to determine the method of aggregation to be applied.
+- `sql`: The SQL used in the `select` clause for this probe.
+
+#### `<project_name>.json`
+
```
Comment (badboy) [inline src/cookbooks/operational_monitoring.md:76]: Some explanation here, something along the lines of:

> The dashboard name, additional information and the list of included objects for this project.

File: src/cookbooks/operational_monitoring.md [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+}
+```
+
+- `name`: Name that will be used as the generated Looker dashboard title.
+- `slug`: The slug associated with the rollout that is being monitored.
+- `channel`: `release`, `beta`, or `nightly`. The channel the rollout is running in.
+- `boolean_pref`: A SQL snippet that results in a boolean representing whether a user is included in the rollout or not (note: if this is not included, the slug is used to check instead).
+- `xaxis`: `submission_date` or `build_id`. specifies the type of monitoring desired as described above.
+- `start_date`: `yyyy-mm-dd`, specifies the oldest submission date or build id to be processed (where build id is cast to date). If not set, defaults to the previous 30 days.
+- `analysis`: An array of objects with the fields `source`, `dimensions`, and `probes` where each object represents data that will be pulled out from a different source.
```
Comment (badboy) [inline src/cookbooks/operational_monitoring.md:109]: Does each object here turn into one tile on the dashboard?
I think that's the thing that's missing to me right now: how does this list correspond to what I see in the end.

Can you link me to the json definitions for the existing Fission dashboard? Then I might be able to better understand this.

File: src/cookbooks/operational_monitoring.md (hunk 2 above)
Comment (emtwo) [reply src/cookbooks/operational_monitoring.md:10]: Yes, I don't have access either. I think you placed that dashboard in the folder, right @scholtzan? Here's what I see:
Screen Shot 2022-01-12 at 12 17 19 PM

If I go to Folders > LookML Dashboards I see them at the bottom of the list.

Comment (emtwo) [reply src/cookbooks/operational_monitoring.md:10]: Thanks!

File: src/cookbooks/operational_monitoring.md (hunk 7 above)
Comment (emtwo) [reply src/cookbooks/operational_monitoring.md:109]: [Here](https://console.cloud.google.com/storage/browser/operational_monitoring;tab=objects?forceOnBucketsSortingFiltering=false&project=emtwo-252813&prefix=&forceOnObjectsSortingFiltering=false) is the bucket with the fission json files

File: src/cookbooks/operational_monitoring.md (hunk 5 above)
Comment (emtwo) [reply src/cookbooks/operational_monitoring.md]: You make some good points! I've updated the location of the explanations.

**Review by scholtzan — APPROVED** [review]

**Pull request merged**